Userland/Libraries/LibGfx/Bitmap.h

```diff
@@ -325,6 +325,12 @@ inline void Bitmap::set_pixel<StorageFormat::BGRA8888>(int x, int y, Color color
     VERIFY(x >= 0 && x < physical_width());
     scanline(y)[x] = color.value(); // drop alpha
 }
+template<>
+inline void Bitmap::set_pixel<StorageFormat::RGBA8888>(int x, int y, Color color)
+{
+    VERIFY(x >= 0 && x < physical_width());
+    scanline(y)[x] = color.value();
```

Comment:
We really need to improve the type safety of color primitives, but let's do that separately from this PR. :^)